client/src/pages/Followers/Followers.tsx

```diff
@@ -0,0 +1,150 @@
+import useStyles from './useStyles';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { useEffect, useState } from 'react';
+import {
+  CssBaseline,
+  Grid,
+  Paper,
+  Tabs,
+  Tab,
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+  ListItemAvatar,
+  Avatar,
+  Button,
+  Box,
+  Typography,
+} from '@material-ui/core';
+import NavBar from '../../components/NavBar/NavBar';
+import { getFollowings, followUser, unfollowUser, getSuggestions } from '../../helpers/APICalls/followers';
+
+export default function Followers(): JSX.Element {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const [tabValue, setTabValue] = useState(0);
+  const [followingsData, setFollowingsData] = useState([]);
+  const [followSugestionsData, setFollowSugestionsData] = useState([]);
+  const [dataChanged, setDataChanged] = useState(false);
+  const handleTabChange = (event: any, newValue: number) => {
+    setTabValue(newValue);
+  };
+  //fetch data
+  useEffect(() => {
+    getFollowingsData().then((data: any) => {
+      setFollowingsList(data);
+    });
+    getFollowSugestionsData().then((data: any) => {
+      setFollowSugestionsList(data);
+    });
+    setDataChanged(false); //reset flag
+  }, [dataChanged]);
```

Comment:
This is not the best way to handle this, it would be better to just update the state based on the response from the update route or update it as part of the same function that makes the API call